webapp/app/actions/usersActions.js

```diff
@@ -117,14 +113,13 @@ export function getUsers(queryParams = DEFAULT_PAGINATION_QUERY) {
         dispatch(getUsersSuccess(usersPayload));
         return usersPayload;
       })
-      .catch(error => dispatch(loadingUsersFailed(error)));
+      .catch(handleErrors);
   };
 }
 
-// TODO move to service errors utility
-function handleErrors(response) {
-  if (response.statusText !== 'OK') {
-    throw Error(response.statusText);
-  }
-  return response;
+function handleErrors(error) {
+  return dispatch =>{
+    logger.error(error.message);
+    dispatch(loadingUsersFailed(error));
+  };
```

Comment:
I think we would need something global for the whole application, can we do that?

Reply:
Yes, what I think is to set up a logging service that will log (and potentially send them to a server) and leave here the dispatch of the fail, which only purpose is to update the store with the error flag to show some feedback.
@mravinale Is that what you are asking?